conf/test.config

```diff
@@ -29,7 +29,7 @@ params {
     protocol     = '10XV2'
 
     // Ignore `--input` as otherwise the parameter validation will throw an error
```

Comment:
Maybe the comment needs to be updated?

Reply:
yeah, I haven't seen any errors here when I removes these params.

Reply:
```suggestion
```